scripts/setup-versions.sh

```diff
@@ -19,7 +19,7 @@
 # The versions should match the declared versions in this file.
 
 # Build dependencies versions.
-FB_OS_VERSION="v2025.04.28.00"
+FB_OS_VERSION="v2025.09.15.00"
```

Comment:
We need recent FBThrift to use the `@thrift.AllowUnsafeOptionalCustomDefaultValue` annotation.